user-manuals/orders/bopis/appnotifications/notifications.md

```diff
@@ -1,25 +1,18 @@
 # Notifications
 
-Upon the arrival of a new order at the store, timely notification is crucial for store associates. BOPIS notifications deliver updates on both new and open orders within a facility, significantly improving the ease of handling order preparation and handover procedures.\
-\
-Store managers can customise their notification preferences for the following topics:
+Timely notifications are crucial for store associates upon the arrival of a new order. BOPIS notifications provide updates on both new and open orders within a facility, significantly improving the efficiency of order preparation and handover procedures. Notifications can be viewed within the BOPIS app by clicking on the `bell` icon on the top right corner of the Orders page.  
 
-* **New Orders:** Receive prompt notifications as new orders become available for preparation at the facility.
+Store managers can customise the following notification preferences. 
+
+* **New Orders:** Receive notifications as new orders become available for preparation at the facility.
```

Comment:
preparation may not be the irght term, fulfillment would be a better word choice

Reply:
Edited.